### ISSUE TITLE
Que: Compatible with APEX 22.2?

### ISSUE THREAD

**Issue by bansi36** [Dani3lSun/apex-plugin-apexsignature#19]
Hello,  Has anyone tested whether this plugin is compatible with latest version of APEX 22.2 or Anyone working on it ?

**Comment by tgarabedyan** [comment]
Yes, it works. But you must create two fuctions, provided in separate sql file manualy. Also, when created region, do not forget to go to "Attributes" and fill missing jQuery fields. Otherwise you will not be able to save the page.
